apps/daimo-mobile/src/view/screen/send/RouteDisplay.tsx

```diff
@@ -0,0 +1,34 @@
+import { ForeignToken, ProposedSwap, amountToDollars } from "@daimo/common";
+import { View, StyleSheet } from "react-native";
+
+import { TextLight } from "../../shared/text";
+
+export function RoutePellet({
+  route,
+  fromCoin,
+  toCoin,
+}: {
+  route: ProposedSwap;
+  fromCoin: ForeignToken;
+  toCoin: ForeignToken;
+}) {
+  const toAmount = amountToDollars(route.toAmount, toCoin.decimals);
+
+  return (
+    <View style={styles.route}>
+      <TextLight style={{ fontSize: 14 }}>
```

Comment:
is this a new text size that's not one of the standard ones in `typography.tsx`?